2023-10-17 08:10:25,390 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2023-10-17 08:10:25,390 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Configure stats pid to 2434965
2023-10-17 08:10:25,390 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2023-10-17 08:10:25,390 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Loading settings from /workspace/TCN/wandb/settings
2023-10-17 08:10:25,390 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-10-17 08:10:25,390 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-10-17 08:10:25,391 INFO    MainThread:2434965 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-10-17 08:10:25,391 INFO    MainThread:2434965 [wandb_init.py:_log_setup():528] Logging user logs to /workspace/TCN/wandb/run-20231017_081025-2ch8wr83/logs/debug.log
2023-10-17 08:10:25,391 INFO    MainThread:2434965 [wandb_init.py:_log_setup():529] Logging internal logs to /workspace/TCN/wandb/run-20231017_081025-2ch8wr83/logs/debug-internal.log
2023-10-17 08:10:25,391 INFO    MainThread:2434965 [wandb_init.py:init():568] calling init triggers
2023-10-17 08:10:25,391 INFO    MainThread:2434965 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'epochs': 200, 'batch_size': 256}
2023-10-17 08:10:25,391 INFO    MainThread:2434965 [wandb_init.py:init():593] re-initializing run, found existing run on stack: a8o4wqhi
2023-10-17 08:10:25,393 INFO    MainThread:2434965 [wandb_run.py:_finish():1934] finishing run dwryu/TCN/a8o4wqhi
2023-10-17 08:10:25,393 INFO    MainThread:2434965 [wandb_run.py:_atexit_cleanup():2168] got exitcode: 0
2023-10-17 08:10:25,393 INFO    MainThread:2434965 [wandb_run.py:_restore():2151] restore
2023-10-17 08:10:25,394 INFO    MainThread:2434965 [wandb_run.py:_restore():2157] restore done
2023-10-17 08:20:24,695 INFO    MainThread:2434965 [wandb_init.py:init():618] starting backend
2023-10-17 08:20:24,695 INFO    MainThread:2434965 [wandb_init.py:init():622] setting up manager
2023-10-17 08:20:24,698 INFO    MainThread:2434965 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-10-17 08:20:24,699 INFO    MainThread:2434965 [wandb_init.py:init():628] backend started and connected
2023-10-17 08:20:24,711 INFO    MainThread:2434965 [wandb_run.py:_label_probe_notebook():1276] probe notebook
2023-10-17 08:20:24,712 INFO    MainThread:2434965 [wandb_run.py:_label_probe_notebook():1286] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-10-17 08:20:24,712 INFO    MainThread:2434965 [wandb_init.py:init():720] updated telemetry
2023-10-17 08:20:24,714 INFO    MainThread:2434965 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
